.github/workflows/build-test-release.yaml

```diff
@@ -97,12 +97,12 @@ jobs:
           egress-policy: 'audit'
 
       # yamllint disable-line rule:line-length
-      - uses: actions/checkout@93cb6efe18208431cddfb8368fd83d5badbf9bfd  # v5.0.1
+      - uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3 # v6.0.0
 
       - name: 'Build Python project'
         id: 'python-build'
         # yamllint disable-line rule:line-length
-        uses: lfreleng-actions/python-build-action@48381cece78a990a6ba93bd5924bcd40bf0d1a7d  # v0.1.20
+        uses: lfreleng-actions/python-build-action@7cff4964bc5d197f0a4411c3e47d4b192ccf19b7 # v0.1.22
```

Comment:
The other issue made me concerned about the accuracy of these other actions. While `actions/checkout` looks good, this SHA is also incorrect:

https://github.com/lfreleng-actions/python-build-action/releases/tag/v0.1.22